contracts/src/v0.8/functions/dev/1_0_0/FunctionsCoordinator.sol

```diff
@@ -125,6 +131,9 @@ contract FunctionsCoordinator is OCR2Base, IFunctionsCoordinator, FunctionsBilli
     address[] memory nodes = this.transmitters();
     bytes[] memory keys = new bytes[](nodes.length);
     for (uint256 i = 0; i < nodes.length; i++) {
+      if (s_nodePublicKeys[nodes[i]].length == 0) {
```

Comment:
@KuphJr @bolekk 
Is reverting here a safe behavior?
The other option would be to not add to `keys` if empty bytes

Reply:
TBH, I prefer empty bytes to a revert, but I think a revert is still safe.

Reply:
Do you mean returning empty bytes within the `keys` returned? Or not adding empty bytes to `keys`?

Reply:
Yeah, why do we need a revert? If somebody gets empty keys, the encryption will fail anyway.